jupyterhub/schema.yaml

```diff
@@ -229,7 +229,7 @@ properties:
           Use this only if you are building your own image & know that a group with this gid
           exists inside the hub container! Advanced feature, handle with care!
 
-          Defaults to 1000, which is the uid of the `jovyan` user that is present in the
+          Defaults to 100, which is the uid of the `jovyan` user that is present in the
```

Comment:
s/uid/gid/

Reply:
I was unfamiliar with the shorthand notation `s/uid/gid/` - based on context I figure I should replace uid with gid now that I look at it!

Reply:
Oh this was actually within the schema for the hub values, so I'll let this remain to 1000-